.github/workflows/ci.yml

```diff
@@ -63,34 +54,14 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Unit tests
-        run: bash scripts/execute_unittests_for_changed_modules.sh
+        run: bash scripts/execute_task_for_changed_modules.sh testDebugUnitTest verifyPaparazziDebug
```

Comment:
This makes it so that we don't need to run the unit tests twice. Since paparazzi only does verification when it's task is on the task graph.

The script is smart enough now to only add the `verifyPaparazziDebug` task to modules that support it. 

Here's an example of what the script would actually run.

```
./gradlew :example:testDebugUnitTest :financial-connections-example:testDebugUnitTest :link:testDebugUnitTest :payments-core:testDebugUnitTest :paymentsheet:testDebugUnitTest :paymentsheet:verifyPaparazziDebug :paymentsheet-example:testDebugUnitTest :financial-connections:testDebugUnitTest :financial-connections:verifyPaparazziDebug
```

Reply:
Amazing.